code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -605,7 +605,11 @@ GLOBAL_VAR(station_nuke_source)
 
 /proc/KillEveryoneOnStation()
 	for(var/mob/living/victim as anything in GLOB.mob_living_list)
-		if(victim.stat != DEAD && is_station_level(victim.z))
+		var/turf/mob = get_turf(victim)
```

Comment:
`var/turf/mob` is an incredibly cursed variable name

Reply:
Is creature ok? I was going for player but it works on all living being.

Reply:
I changed it to living_target for keeping it descriptive. I hope it is okay. Sorry for the trouble.

Reply:
I was thinking something like `target_turf`, since it's the... turf, of the target / victim

Reply:
Okie! Done.